test-reqs.txt

```diff
@@ -16,5 +17,6 @@ snowballstemmer>=1.1
 sphinx_rtd_theme
 sphinxcontrib-websupport
 sqlalchemy>=0.9
+tox
```

Comment:
I'm not a fan of this - we want to be able to reference `test-reqs.txt` from the `reqs` section in `tox.ini`, as seen in #4120. You should either already have `tox` installed, or you should install dependencies via `pip` yourself, e.g.

    $ cd sphinx
    $ pip install .[test]